packages/remote-logs/src/transports.ts

```diff
@@ -58,29 +59,22 @@ export const lokiTransport: transportFunctionType = (props: LokiTransportProps)
   const [credentials, href] = lokiUrl.split('@')
   const [username, password] = credentials.split('//')[1].split(':')
   const protocol = credentials.split('//')[0]
-  const axiosConfig: AxiosRequestConfig = {
-    method: 'post',
-    url: lokiUrl,
-    data: payload,
+  const authHeader = `Basic ${Buffer.from(`${username}:${password}`).toString('base64')}`
+  const config = {
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: authHeader,
+    },
   }
 
-  // If username and password are present
-  // in the URL, use them for auth
-  if (username && password) {
-    axiosConfig.auth = {
-      username,
-      password,
-    }
-    axiosConfig.url = `${protocol}//${href}`
-  }
-
-  axios(axiosConfig)
+  axios
+    .post(`${protocol}//${href}`, payload, config)
     .then((res) => {
       if (res.status !== 204) {
-        throw new Error(`Expected Loki to return 204, received ${res.status}`)
+        console.warn(`Expected Loki to return 204, received ${res.status}`)
```

Comment:
Did you intend to leave `console.warn()` instead of exception?

Reply:
If I leave a throw/exception in here then you need to wrap log in a try/catch which seems heavy handed. If you don't you end up with "Unhandled promise rejection". I could do nothing but then one would never know there is a problem and since this is the logger I can't really use logger to log an error 😕

Reply:
That is fine. Just wanted to to double check it was intentional. 👍